data-service/src/main/java/uk/gov/gchq/palisade/service/data/exception/ErrorDetails.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.data.exception;
+
+import uk.gov.gchq.palisade.ToStringBuilder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class ErrorDetails {
+
+    private Date date;
```

Comment:
java.util.Date is legacy. Upgrade to something from java.time. Probably a ZonedDateTime

Reply:
java.util.Date has been removed. Now uses java.time.ZonedDateTime.